cmd/jsonreporter.go

```diff
@@ -30,20 +30,21 @@ type histogramPoint struct {
 }
 
 type jsonResult struct {
-	TotalRequests            int64            `json:"totalRequests"`
-	TotalSuccessCodes        int64            `json:"totalSuccessCodes"`
-	TotalErrors              int64            `json:"totalErrors"`
-	TotalIDmismatch          int64            `json:"TotalIDmismatch"`
-	TotalTruncatedResponses  int64            `json:"totalTruncatedResponses"`
-	ResponseRcodes           map[string]int64 `json:"responseRcodes,omitempty"`
-	QuestionTypes            map[string]int64 `json:"questionTypes"`
-	QueriesPerSecond         float64          `json:"queriesPerSecond"`
-	BenchmarkDurationSeconds float64          `json:"benchmarkDurationSeconds"`
-	LatencyStats             latencyStats     `json:"latencyStats"`
-	LatencyDistribution      []histogramPoint `json:"latencyDistribution,omitempty"`
+	TotalRequests             int64            `json:"totalRequests"`
+	TotalSuccessCodes         int64            `json:"totalSuccessCodes"`
+	TotalErrors               int64            `json:"totalErrors"`
+	TotalIDmismatch           int64            `json:"TotalIDmismatch"`
+	TotalTruncatedResponses   int64            `json:"totalTruncatedResponses"`
+	ResponseRcodes            map[string]int64 `json:"responseRcodes,omitempty"`
+	QuestionTypes             map[string]int64 `json:"questionTypes"`
+	QueriesPerSecond          float64          `json:"queriesPerSecond"`
+	BenchmarkDurationSeconds  float64          `json:"benchmarkDurationSeconds"`
+	LatencyStats              latencyStats     `json:"latencyStats"`
+	LatencyDistribution       []histogramPoint `json:"latencyDistribution,omitempty"`
+	TotalDNSSECSecuredDomains *int             `json:"totalDNSSECSecuredDomains,omitempty"`
 }
 
-func (s *jsonReporter) print(w io.Writer, b *Benchmark, timings *hdrhistogram.Histogram, codeTotals map[int]int64, totalCounters Counters, qtypeTotals map[string]int64, topErrs orderedMap, t time.Duration) error {
+func (s *jsonReporter) print(w io.Writer, b *Benchmark, timings *hdrhistogram.Histogram, codeTotals map[int]int64, totalCounters Counters, qtypeTotals map[string]int64, topErrs orderedMap, t time.Duration, authenticatedDomains map[string]struct{}) error {
```

Comment:
improve `standardReporter.print` and `jsonReporter.print` by introducing parameter object?